src/iobuf.rs

```diff
@@ -423,7 +423,7 @@ pub trait Iobuf: Clone + Show {
   ///
   ///     let mut sum = 0u16;
   ///
-  ///     for _ in range(0, num_shorts) {
+  ///     for _ in (0 .. num_shorts) {
```

Comment:
Normally, I don't think so, but I think there was a parsing error without them in this case, can't say why.